frontend/packages/console-shared/src/hooks/flag.ts

```diff
@@ -1,9 +1,14 @@
 // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
can you try to move the implementation to sdk package ? I think it may be quite easy, looking at the size